app/views/groups/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1 class="text-center text-break">
+  <%= render GroupNameComponent.new(group: @group) %>
+</h1>
+
+<div class="text-end">
+  <%= mail_to @admin.email, t(:contact_group), class: "btn btn-sm btn-primary" %>
```

Comment:
This fails if the group has no admin (e.g. group `everyone`).

@Encotric What should be the expected behavior in such cases?

Reply:
As stated in #276, the group `everyone` is not shown to the user to be clicked on.

Should we catch this error anyway? E.g., in controller when `@admin` is `null`?

Reply:
should resolve with [9035a27](https://github.com/hpi-swt2/bookkeeper-portal-red/pull/360/commits/9035a272508245be967d98d9770512e3ffa7e569)